Doctrine/Orm/Filter/DateFilter.php

```diff
@@ -175,7 +175,11 @@ public function getDescription(ResourceInterface $resource)
         }
 
         foreach ($properties as $property => $nullManagement) {
-            if (!$this->isPropertyMapped($property, $resource)) {
+            if (
+                !$this->isPropertyEnabled($property)
+                || !$this->isPropertyMapped($property, $resource)
```

Comment:
Also, `isPropertyEnabled` check is redundant as it will always return `true` here.